lib/ansible/modules/service_facts.py

```diff
@@ -70,7 +70,7 @@
         state:
           description:
           - State of the service.
-          - 'This commonly includes (but is not limited to) the following: C(failed), C(running), C(stopped) or C(unknown).'
+          - 'This commonly includes (but is not limited to) the following: C(running), C(failed), C(dead), C(exited), C(running), C(stopped) or C(unknown).'
```

Comment:
```suggestion
          - 'This commonly includes (but is not limited to) the following: C(running), C(failed), C(dead), C(exited), C(stopped) or C(unknown).'
```
running appears twice